docs/react-testing-library/example-intro.md

```diff
@@ -20,20 +20,20 @@ jest.mock('axios')
 
 test('loads and displays greeting', async () => {
   const url = '/greeting'
-  const { getByText, getByRole } = render(<Fetch url={url} />)
+  const { container, asFragment } = render(<Fetch url={url} />)
```

Comment:
Looks like c&p from below or why do we need these now?

Reply:
Oh yes, it definitely was. I'll remove them.